spec/services/badge_granter_spec.rb

```diff
@@ -482,14 +482,17 @@
       expect(UserBadge.find_by(user_id: user.id, badge_id: Badge::GreatPost)).to eq(nil)
     end
 
-    it 'calls user_badge_granted on backfilled badge' do
+    it "calls 'user_badge_granted' DiscourseEvent on backfilled badge" do
```

Comment:
`sends` or `triggers` is maybe a bit better than `calls` when referring to events, wdyt?

Reply:
Ah good point let me update the code.